index.mjs

```diff
@@ -4,6 +4,9 @@ import cp from 'child_process'
 import os from 'os'
 import util from 'util'
 import _fs from 'fs'
+import sudo from 'sudo-prompt'
+
+const __dirname = path.dirname(new URL(import.meta.url).pathname);
```

Comment:
I'm afraid `import.meta` wont work because the ESM is being transpiled to CJS. Would `process.cwd()` be enough instead of recreating `__dirname`?